openapi.yml

```diff
@@ -0,0 +1,506 @@
+openapi: 3.0.0
+info:
+  description: Workflow Server HTTP API
+  version: 1.0.0
+  title: Workflow Server HTTP API
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://workflow-service-prod.stanford.edu'
+    description: Production service
+    variables:
+      env:
+        default: prod
+  - url: 'https://workflow-service-stage.stanford.edu'
```

Comment:
Should lines 10 and 15 use `{env}` instead like https://github.com/sul-dlss/dor-services-app/blob/master/openapi.yml#L10-L15 ?

Reply:
Ah, I see it now. I can update that.